pkg/pg/txdb_test.go

```diff
@@ -0,0 +1,69 @@
+package pg
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	_ "github.com/marcboeker/go-duckdb"
+
+	"github.com/google/uuid"
+	"github.com/jmoiron/sqlx"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"context"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/utils/tests"
+)
+
+func TestTxDBDriver(t *testing.T) {
+	dbURL, ok := os.LookupEnv("CL_DATABASE_URL")
+	if !ok {
+		fmt.Sprintf("CL_DATABASE_URL not set--falling back to testing txdb backed by an in-memory db")
```

Comment:
Did you mean `t.Log`?
```suggestion
		t.Log("CL_DATABASE_URL not set--falling back to testing txdb backed by an in-memory db")
```